books/free-programming-books-pt_BR.md

```diff
@@ -267,14 +266,14 @@
 
 #### Meteor
 
-* [Tudo sobre Meteor](http://udgwebdev.com/meteor/)
+* [Tudo sobre Meteor](https://udgwebdev.github.io/meteor/)
```

Comment:
LGTM since seems a hostname expiration

last archive: https://web.archive.org/web/20191002181424/http://udgwebdev.com/meteor/

Reply:
Looks like it's no longer free - what's left is not a book. Probably related to the leanpub change: #6852 Should probably remove it.